Spatial.gd

```diff
@@ -833,28 +854,28 @@ func _on_NodeEditorDialog_hide():
 
 func _on_DeleteNode_pressed():
 	if self.currently_selected_node.point_type == "icon":
```

Comment:
Can we have logic that does not rely on `point_type` branching in multiple places?